element.go

```diff
@@ -147,8 +148,10 @@ func NewElement(t tag.Tag, data interface{}) (*Element, error) {
 	if err != nil {
 		return nil, err
 	}
-	rawVR := tagInfo.VR
-
+	rawVR := strings.Split(tagInfo.VR, tag.MultipleVRsAllowedSeparator)[0]
+	if t == tag.PixelData {
+		rawVR = "OW"
+	}
```

Comment:
Ugly but I had to do this to preserve previous behaviour. IMO, we should deprecate this `NewElement` in favour of something that directly specifies the VR. Guessing the VR is too hard.

Reply:
No problem, I think this is fine for now. In a future change, we can make a breaking API change to pass a VR. For this next batch of larger changes, I expect there will be API breakages but they will be for good reasons so it's worth it imo!